code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -409,6 +409,7 @@
 	name = "Stunning Aura"
 	color = RUNE_COLOR_RED
 	invocation = "Fuu ma'jin."
+	var/ratvarscum_punish = 10
```

Comment:
more serious/at a glance and utilitarian variable names, same for the other places.